doc/source/reference/c-api.array.rst

```diff
@@ -1671,11 +1671,13 @@ Conversion
 .. c:function:: PyObject* PyArray_GetField( \
         PyArrayObject* self, PyArray_Descr* dtype, int offset)
 
-    Equivalent to :meth:`ndarray.getfield<numpy.ndarray.getfield>` (*self*, *dtype*, *offset*). Return
-    a new array of the given *dtype* using the data in the current
-    array at a specified *offset* in bytes. The *offset* plus the
-    itemsize of the new array type must be less than *self*
-    ->descr->elsize or an error is raised. The same shape and strides
+    Equivalent to :meth:`ndarray.getfield<numpy.ndarray.getfield>`
+    (*self*, *dtype*, *offset*). This function `steals a reference
+    <https://docs.python.org/3/c-api/intro.html?reference-count-details>`
+    to `PyArray_Descr` and returns a new array of the given `dtype` using
+    the data in the current array at a specified `offset` in bytes. The
+    `offset` plus the itemsize of the new array type must be less than `self
+    ->descr->elsize` or an error is raised. The same shape and strides
     as the original array are used. Therefore, this function has the
     effect of returning a field from a structured array. But, it can also
```

Comment:
You can see the rendered document [here](https://7400-908607-gh.circle-artifacts.com/0/home/circleci/repo/doc/build/html/reference/c-api.array.html#c.PyArray_GetField). I think you need a single underline `_` at the end of line 1676 (after the backtick) to turn it into a link. 

double back-ticks are needed to format `self->descr->elsize` as code.

Reply:
@mattip Updated.